assets/src/edit-story/components/panels/panel/useKeyboardHandlers.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import {
+	useCallback,
+	useLayoutEffect,
+} from '@wordpress/element';
+
+const KEY_UP = 38;
+const KEY_DOWN = 40;
+const DELTA_CHANGE = 20; // change in pixels when pressing arrow keys
+
+function useKeyboardHandlers( handle, handleHeightChange ) {
+	// Handle up/down keypresses to move separator.
+	// TODO Should be rewritten to use MouseTrap when added to .
+	const handleKeyPress = useCallback( ( evt ) => {
+		if ( [ KEY_UP, KEY_DOWN ].includes( evt.keyCode ) ) {
+			const direction = evt.keyCode === KEY_UP ? 1 : -1;
+			handleHeightChange( direction * DELTA_CHANGE );
+			evt.stopPropagation();
+			evt.preventDefault();
+		}
+	}, [ handleHeightChange ] );
+
+	// On initial render assign keyboard listener to handle up/down arrow presses.
+	useLayoutEffect( () => {
+		const element = handle.current;
+		element.addEventListener( 'keydown', handleKeyPress );
+
+		return () => {
+			if ( element ) {
```

Comment:
Why do we have to check if element exists here?

Reply:
I had some issues at some point with "element being undefined", but it appears they're not relevant anymore. Fixed in 6283157.